README.md

```diff
@@ -4,62 +4,48 @@ JavaScript package to share frontend UI components, styles, and configuration.
 
-3. Finally (not positive this is necessary) rerun `yarn install` _in both directories_ to make sure all necessary dependencies are installed and linked properly.
+3. Finally (not positive this is necessary) rerun `yarn install`
 
 That's it! You're all configured.
 
 ## _Doing_ Local Development
 
 Once the steps from above have been completed, to actually make changes you'll want to:
 
-0. **In the Rails app** (e.g. `OS`): Start the Rails server and whatever script needs to be running to compile assets (`bin/webpack-dev-server` in older apps, probably something like `yarn build --watch` and `yarn build:css --watch` for Rails 7+).
+0. **In the Rails app** (e.g. `OS`): Start the Rails server and whatever script needs to be running to compile assets (probably something like `yarn build --watch` and `yarn build:css --watch` for Rails 7+).
 
-1. **In this directory**: Start the compilation script with `yarn compile:watch` (only needed for JS changes)
+1. Now make a change in this repo and it should show up on the next page refresh in the consuming app (e.g. OS).
 
-2. Now make a change in this repo and it should show up "immediately" (after compilation completes) in the consuming app (e.g. OS).
-
-    * NOTE: For Rails 6 apps you don't even need a browser reload, as some fast-refresh config appears to be triggering it for us (although that can be a bit delayed if the recompilation takes a while... which, in my testing, it tends to). Leadership is on Rails 7 and it appears to require a manual browser refresh to pick up changes, but no server restarts are required as long as the various JS and CSS watcher commands are running.
+#### Details
 
+* **Changes to SCSS files** are picked up immediately by the Rails app as long as you have `yarn build:css --watch` running.
 
-#### Details
+* **Changes to JS files** are picked up immediately by the Rails app as long as you have `yarn build --watch` running.
 
-* **Changes to SCSS files** (loaded directly, no SCSS->CSS compilation needed within shared-ui) are picked up immediately by the Rails app as long as you have `bin/webpack-dev-server` running.
+* **Changes to `tailwind.config.js`** _do not_ hotreload -- changes there require a restart of the Rails app's compiler.
 
-* **Changes to JS files** _are also_ picked up immediately, as long as you're running `yarn compile:watch` in this directory.
+## Caveats
 
-* **Changes to `tailwind.config.js`** _do not_ hotreload -- changes there require a restart of the Rails app's compiler (e.g. `bin/webpack-dev-server`).
+Super annoyingly, as of Yarn v3 the `yarn link` command adds a `resolutions` key directly to `package.json`... which we obviously can't commit to production, since it's only a valid path on your local computer.  I WOULD LOVE A WORKAROUND HERE IF ANYONE HAS ONE! In the meantime, I've updated CI for all our consuming apps to include a linter step that will fail if you accidentally leave in the resolutions key.
```

Comment:
I was looking at threads around this, and one other option might be to automatically do a `yarn unlink` as part of the commit process, and then run `yarn link` again when it's done. Obviously a hack, but might work.